core/system/src/unvic.c

```diff
@@ -583,6 +583,18 @@ void unvic_init(void)
      * levels. */
     assert(__UVISOR_NVIC_MIN_PRIORITY < UVISOR_VIRQ_MAX_PRIORITY);
 
+    /* Set the priority of each exception. SVC is lower priority than
+     * MemManage, BusFault, and UsageFault, so that we can recovery from
+     * stacking MemManage faults more simply. Set DebugMonitor, PendSV, SYSTICK
+     * to the same priority as SVC. */
+    NVIC_SetPriority(MemoryManagement_IRQn, 0);
```

Comment:
Not sure if there's an elegant way to do this, but it seems to me that these hard-coded priority values are too decoupled from the `__UVISOR_NVIC_MIN_PRIORITY ` symbol. For example, at least we should check that the highest priority number used here (`1`) is lower than the `__UVISOR_NVIC_MIN_PRIORITY` value.